samples/sensor/veaa_x_3/src/main.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2024 Vitrolife A/S
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/kernel.h>
+#include <zephyr/sys/printk.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/drivers/sensor/veaa_x_3.h>
+
+static const struct device *const dev = DEVICE_DT_GET_ONE(festo_veaa_x_3);
```

Comment:
perhaps use an alias for this?

Reply:
`..GET_ONE` is more common in `samples/sensor` dir.

Reply:
Okay, well, I think maybe an alias would be better, but would like @MaureenHelm's opinion since I don't know why the GET_ONE is the standard here if it is

Reply:
Alright. I just did a recursive grep and got about 2-3 more matches on GET_ONE.

Reply:
Is this the only thing that remains before you can approve @decsny?

Reply:
I'm not blocking this, I think the sensors and dac maintainers should review now